frontend/src/components/users/ReviewsContainer.vue

```diff
@@ -34,26 +34,34 @@ export default {
         ReviewsContainerItem
     },
     computed: {
-        ...mapState('place', {
+        ...mapState('users', {
             places: 'places',
         }),
         ...mapGetters({
             userProfile: 'users/getUserProfile'
         }),
-        ...mapGetters('place', ['getReviewsById' , 'getUserReviewsAll']),
+        ...mapGetters('users', ['getReviewsById' , 'getUserReviewsAll']),
         filteredUsersPlaces: function () {
             return this.getReviewsById(parseInt(this.$route.params.id));
         },
         filterAllReviewUser: function () {
             return this.getUserReviewsAll(parseInt(this.$route.params.id)).length;
         },
     },
+    methods: {
+        ...mapActions({
+            fetchAllPlaces: 'users/fetchAllPlaces'
+        })
+    },
     created() {
-        this.$store.dispatch('place/fetchPlaces', this.$route.query);
+        this.$store.dispatch('users/fetchAllPlaces')
+            .then(()=>{
+                this.$parent.$data.isLoading = false;
```

Comment:
it is not a good approach to change parent state, it is better to emit custom event "onLoad" and in the parent component do what you wish